### DISCUSSION TITLE
go: github.com/Antonboom/errname@v0.1.5 requires golang.org/x/sys@v0.0.0-20210915083310-ed5796bab164: missing go.sum entry; to add it: go mod download golang.org/x/sys

### DISCUSSION THREAD

**Discussion by Activity00** [golangci/golangci-lint#2955] [Q&A] [answered]
go: github.com/Antonboom/errname@v0.1.5 requires
golang.org/x/sys@v0.0.0-20210915083310-ed5796bab164: missing go.sum entry; to add it:
go mod download golang.org/x/sys

but i try to execute go mod download golang.org/x/sys the same error

**Comment by Antonboom** [comment, marked as answer]
Please, provide version of used `golangci-lint` and what command you did.
Looks like too old problem – `golangci-lint@v1.54.2` depends on `errname@v0.1.12` and `golang.org/x/sys v0.12.0`.
